tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -456,13 +491,12 @@ private MockTracerAgent GetMockAgent()
     {
         var mockAgent = EnvironmentHelper.GetMockAgent();
 
-        if (mockAgent is not MockTracerAgent.TcpUdpAgent agent)
+        if (mockAgent is MockTracerAgent.TcpUdpAgent agent)
         {
-            throw new NotSupportedException($"Expected the mock agent to be of type {typeof(MockTracerAgent.TcpUdpAgent)} but found {mockAgent.GetType()}.");
+            SetEnvironmentVariable(ConfigurationKeys.AgentPort, agent.Port.ToString());
```

Comment:
I don't think this is necessary - the `EnvironmentHelper` should set that automatically, but I could be wrong! 🙂

Reply:
No, I don't think this is happening, at least I didn't find it. 🤔

Reply:
Ah, we do all this in `TestHelper.StartSample()` which takes care of setting all the profiler variables etc, which it looks like `DebuggerTestHelper` calls into, so that should do it?

Reply:
Oh, you are right.
I removed redundant code at 99f4b845d17ec9a7e13c11979b30502e7112ba12